signer/Cargo.toml

```diff
@@ -9,6 +9,7 @@ name = "relay-server"
 
 [[example]]
 name = "relay-client"
+required-features = ["testing"]
```

Comment:
Yeah, totally isn't. I wanted to remove the `testing` feature from the default features but didn't follow through since I was unsure if that was truly better.

Reply:
I went with removing it `testing` from being a default feature.

Reply:
And I reverted that change 😄. It's harder, if not impossible, to get it the way I'd like it to be.

Reply:
Yeah no I think I could go either way on that. The advantage of having it as a default feature is that we don't get a bunch of code marked as inactive in our IDEs (though that can be fixed by configuring rust-analyzer).